res/skins/Tango/style.qss

```diff
@@ -1560,8 +1560,8 @@ decks, samplers, mic, aux, fx */
       /* Image is rendered correctly but size of the tick mark containers
       won't change. Also, only with this option the hover bg color defined
       above will be applied... it's qt magic
-      image: url(skin:/buttons/btn_fx_selector_tick.svg) no-repeat center center; */
-      background: #0081B7;
+      image: url(skin:/buttons/btn_fx_selector_tick.svg) no-repeat center center;
+      background: #0081B7; */
     }
```

Comment:
I'll keep all those comments. Helpful, also it took quite a while to gather those infos..